### PR TITLE
BUILD: CMake usability enhancements

### PR DESCRIPTION
**Pull request by Krzmbrzl**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -77,6 +77,15 @@ endif()
 include(compiler)
 include(os)
 
+message(STATUS "##################################################")
+message(STATUS "Building Mumble version: ${version}")
+if(64_BIT)
+	message(STATUS "Architecture: 64bit (x64)")
+else()
+	message(STATUS "Architecture: 32bit (x86)")
+endif()
```
Comment (davidebeatrici) [inline CMakeLists.txt:86]: The architecture is not guaranteed to be `x86`, it could be `ARM` for example.

I would just print `64 bit` or `32 bit`.

Comment (Krzmbrzl) [reply CMakeLists.txt:86]: Ah I always thought those would be synonymous ^^

Comment (davidebeatrici) [reply CMakeLists.txt:86]: `x64` can indeed be misleading.

Comment (Krzmbrzl) [reply CMakeLists.txt:86]: Fixed in #4447

**Pull request merged**